changelog/v1.19.0-beta3/doc-fixes.yaml

```diff
@@ -0,0 +1,5 @@
+changelog:
+  - type: NON_USER_FACING
+    description: >-
+      Weekly doc fixes, including API proto doc format changes and an update to the docs copy workflow.
+      skipCI-kube-tests:true
```

Comment:
might need to set this to false just in case to make sure the proto changes all pass the tests